public/style.css

```diff
@@ -1,20 +1,40 @@
-/*app layout*/
+html{
+  height:100%;
+  line-height: 0.8;
+}
 
-.container {
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  align-items: center;
+body{
+  margin: 0;
+  padding: 0;
+  height: 100%;
 }
 
-.position{
-  position: fixed;
+.pl1p5{
+  padding-left:2.5em;
+}
+.pr1p5{
+  padding-right:1.5rem;
+}
+/*revisit after merging nogas PRs*/
```

Comment:
Its ready. the comments are for me to do when we merge nogas PRs and mine. I dont wanna clean the css file before knowing why things are written in it